src/WebSdk/Publish/Targets/Microsoft.NET.Sdk.Publish.targets

```diff
@@ -30,6 +30,12 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     <_DotNetCLIToolTargetsDir Condition=" '$(_DotNetCLIToolTargetsDir)'=='' ">$(MSBuildThisFileDirectory)DotNetCLIToolTargets\</_DotNetCLIToolTargetsDir>
     <_PublishProfilesDir Condition=" '$(_PublishProfilesDir)'=='' ">$(MSBuildThisFileDirectory)PublishProfiles\</_PublishProfilesDir>
 
+    <PublishIISAssets Condition=" '$(TargetFrameworkIdentifier)' == '.NETCoreApp' 
```

Comment:
Would it make sense to evaluate this in the target? That way you do not necessarily rely on import order.

Reply:
This is currently used in 3 different targets. So, I was trying to reduce duplication. And property seemed safer if someone wants to use this later in an item group for example.